keylogger-mvp/exec/exec.py

```diff
@@ -1,6 +1,25 @@
-def divisor_list(n):
-    return [1,2,3,6]
+def fizzbuzz(n):
+    if n % 3 == 0 and n % 5 == 0:
+        return "FizzBuzz"
+    elif n % 5 == 0:
+        return "Fizz"
+    elif n % 3 == 0:
+        return "Buzz"
+    else:
+        return ""
+
+
+    # divisors = []
+    # for i in range(1, n + 1):
+    #     if n % i == 0:
```

Comment:
remove commented out code?

Reply:
This is code written during testing and I have added it to gitignore, not sure whether I should remove or not

Reply:
should almost always remove commented out code, I would encourage it.

Reply:
Yeah, I meant that this is generated code not code that does anything. Should it all be removed and the file just be empty?